external-docs/docs/quick-start/qube config.md

```diff
@@ -0,0 +1,253 @@
+# `qube-config.json`
+
+The `qube-config.json` file is a mechanism for expressing the configuration of a CSV-W for use by csvcubed's methods and command line interface. It is comprised of two main parts, the catalogue information (i.e. metadata) and the column definitions. Users naming columns in their `.csv` files with reserved names allows for producing RDF Cube Vocabulary-targetted CSV-W with less configuration.
+
+## Qube configuration overview
+
+There are four steps, some of which are optional for generating a CSV-W using csvcubed. Depending whether the input `.csv` file is configured to use conventional column names, a compainion `qube-config.json` may not even be required.
+
+
+
+```mermaid
+graph LR
+	subgraph "Fastest Qube"
```

Comment:
![Screenshot 2022-01-31 at 16 32 21](https://user-images.githubusercontent.com/73846016/151833806-c0bc9824-9176-4767-a7f4-1395fb5e20e4.png)

Reply:
Are we really coping with custom RDF at this point in time?

Reply:
No. I can grey it out or remove.

Reply:
It'd be great if you could remove it for now, please.